packages/embark/src/lib/core/processes/processManager.js

```diff
@@ -65,6 +69,42 @@ class ProcessManager {
     return processList;
   }
 
+  _registerCommands() {
+    // do not allow whisper service to be started/stopped as it requires a restart of embark
+    const availableProcesses = Object.keys(this.processes).filter((name) => !["whisper", "embark"].includes(name.toLowerCase()));
+    this.plugin.registerConsoleCommand({
+      description: __(`Starts/stops the process. Options: ${availableProcesses.join(", ")}`),
+      matches: (cmd) => {
+        return availableProcesses.some((name) => {
+          name = name.toLowerCase();
+          return [`service ${name} on`, `service ${name} off`].includes(cmd.toLowerCase());
+        });
+      },
+      usage: `service [process] on/off`,
+      process: (cmd, callback) => {
+        const enable = cmd.trim().endsWith('on');
+        const matches = cmd.match(/^service[\s](.*)[\s](?:on|off)$/) || [];
+        const name = matches[1];
+        this.logger.info(`${enable ? 'Starting' :  'Stopping'} the ${name} process...`);
+        if(enable) {
```

Comment:
I think CI will fail here due to missing spaces

Reply:
Do you mean `if (enable) {`? 

I think that's only a rule in `tslint`, not `eslint`.